src/pages/my_page/user_profile/MyProfile.tsx

```diff
@@ -69,145 +87,296 @@ export const MyProfile = () => {
+                        {t('common.email')}
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={14} md={12}>
+                      <Field name={'contactDetails.email'}>
+                        {({ field }: FieldProps<string>) => (
+                          <TextField
+                            {...field}
+                            type="email"
+                            data-testid={dataTestId.myPage.myProfile.emailField}
+                            label={t('common.email')}
+                            fullWidth
+                            size="small"
+                            variant="filled"
+                          />
+                        )}
+                      </Field>
+                    </Grid>
+                    <Grid item xs={16} md={3}>
+                      <Typography fontWeight="bold" sx={typographyStyling}>
+                        {t('my_page.my_profile.personal_web_page')}
+                      </Typography>
+                    </Grid>
+                    <Grid item xs={14} md={12}>
+                      <Field name={'contactDetails.webPage'}>
+                        {({ field }: FieldProps<string>) => (
+                          <TextField
+                            {...field}
+                            type="url"
+                            fullWidth
+                            data-testid={dataTestId.myPage.myProfile.webPageField}
+                            label={t('my_page.my_profile.personal_web_page')}
+                            size="small"
+                            variant="filled"
+                          />
+                        )}
+                      </Field>
+                    </Grid>
+                  </Grid>
+                  <Grid container>
+                    <Grid item xs={16}>
+                      <Box
+                        sx={{
+                          height: '100%',
+                          display: 'flex',
+                          flexDirection: 'column',
+                          justifyContent: { xs: 'center', md: 'flex-start' },
+                          alignItems: 'center',
+                        }}>
+                        <Typography variant="h3" sx={{ mb: '1rem' }}>
+                          {t('my_page.my_profile.profile_picture')}
+                        </Typography>
+                        <ProfilePictureUploader personId={personId} />
+                      </Box>
+                    </Grid>
+                  </Grid>
+                </Box>
 
-        <Box sx={{ gridArea: 'research-profile' }}>
-          <ResearchProfilePanel person={person} isLoadingPerson={personQuery.isLoading} />
-        </Box>
+                <Grid container sx={{ bgcolor: 'secondary.dark', py: '1rem', mt: '1rem' }}>
+                  <Grid item md={5} sx={{ display: 'flex', justifyContent: 'center' }}>
+                    <LoadingButton
```

Comment:
disse sliter litt på mindre skjermed

![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/f55d67a2-5158-4c84-b74c-418081ad4f73)
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/bd6e1260-c136-46e5-b2ea-6a8d9fae58e6)

Reply:
"Lagre og se" fremstår som en alternativ mulighet, siden "Lagre" er filled. Men den alternative muligheten har en mye større knapp nå 🤔 
Har ikke sett skissene for dette, men syns det virker litt rart?
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/3ac49de1-d87c-4dac-bad3-2251c182e8cb)

Reply:
avbryt og lagre  ser ut til å være sentrert på mindre skjermer, men ikke større. Men vet ikke om det er by design eller ikke

Reply:
Hadde glemt å legge inn en margin-break for medium skjerm, derfor knappen ble større 😬

Reply:
Dette ordnet seg selv når "Lage og se min forskerprofil"-knappen ble fjernet